src/main/java/com/thebois/models/inventory/Inventory.java

```diff
@@ -9,7 +9,7 @@
 /**
  * Base implementation for all inventories.
  */
-public abstract class AbstractInventory {
+public class Inventory {
```

Comment:
Don't know if it's necessary, but maybe we should have it implement some kind of `IInventory` interface to reduce coupling?

Reply:
Yes, already on from @JacobBredin review 👍